doc and changelog update (#1128)
pimutils/khal: README.rst

Documentation
-------------
For khal's documentation have a look at readthedocs_.

.. _readthedocs: http://khal.readthedocs.org/

Installation
------------
khal is packaged for most `operating systems`__ and should be installable with
your standard package manager.

.. __: https://repology.org/project/python:khal/versions

> For some exemplary OS you can find installation instructions below. Otherwise
> see the documentation_ for more information.

Reviewer: One of these might read better. I'm leaning to "platforms" because of pip.

```suggestion
You can find installation instructions for some platforms below. Otherwise see
the documentation_ for more information.
```
```suggestion
You can find installation instructions for some operating systems below.
Otherwise see the documentation_ for more information.
```

.. _documentation: https://khal.readthedocs.io/en/latest/install.html

Debian/Ubuntu
~~~~~~~~~~~~~

    apt install khal

Nix
~~~

    nix-env -i khal

Arch
~~~~

>     pacman -s khal

Reviewer 2: ```suggestion
    pacman -S khal
```

Not being pedantic: `-s` is something different.

Author: shows I never used arch

Brew
~~~~

    brew install khal

FreeBSD
~~~~~~~

    pkg install py-khal

> 

Reviewer: Is it worth including the release version from pip too? I'm not sure of the answer.

```suggestion

Pip
~~~
    pip install khal

```

> Install latest version

Reviewer: ```suggestion
Install latest (unreleased) version
```
Just to clarify?
